tests/nat/conftest.py

```diff
@@ -182,8 +182,14 @@ def apply_global_nat_config(duthost, config_nat_feature_enabled):
     if 'nat' not in status or status['nat'] == 'disabled':
         pytest.skip('nat feature is not enabled with image version {}'.format(duthost.os_version))
 
+    # Clear all nat rules in iptables
+    duthost.command("sudo iptables -F -t nat")
```

Comment:
What the motivation of these changes?

Reply:
In sonic-swss/cfgmgr/natmgr.cpp, the NAT rules will not be flushed after the natmgrd restart.
So if other test case doesn't disable nat feature normally (like test_nat_dynamic_enable_disable_nat_docker), the iptables rules will have residue.